starter-code/src/app/movies.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Movie, movieList } from './movies-data';
+
+@Injectable()
+export class MoviesService {
+  
+  movies: Array<Movie> = [...movieList];
```

Comment:
Is there a particular reason to create a copy of the array here?
Why not just `movies: Array<Movie> = movieList`?
This is cool ES6 but essentially it's doing  an extra `movieList.length` computations :)

Reply:
Unless it didn't work for some reason?